src/olympia/stats/models.py

```diff
@@ -41,8 +41,12 @@ class StatsSearchMixin(SearchMixin):
 
 class CollectionCount(StatsSearchMixin, models.Model):
     collection = models.ForeignKey('bandwagon.Collection')
-    count = models.PositiveIntegerField()
-    date = models.DateField()
+
+    # index name in database: `count`
```

Comment:
Everytime you mention `in database` it should read `in dev/stage/prod database` or something, just to remove any confusion.